### PR TITLE
zookeeper: parameterized image and imageTag

### PR DESCRIPTION
**Pull request by squ1d123**
**What this PR does / why we need it**:
Allows the user to specify the zookeeper images as a parameter.
This is required for people trying to run this helm chart in a air gapped environment.
Default is left as current behaviour.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*:
Lets users workaround issue #4540

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by squ1d123** [comment]
Hey can you please review this? 
/assign @kow3ns

File: incubator/zookeeper/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: zookeeper
 home: https://zookeeper.apache.org/
-version: 0.6.3
+version: 0.6.4
```
Comment (timstoop) [inline incubator/zookeeper/Chart.yaml]: This adds a new feature, please bump the minor part of the version.

**Review by timstoop — CHANGES_REQUESTED** [review]

**Comment by cmarquezrusso** [comment]
+1

**Comment by tedmiston** [comment]
+1 to this idea

I think we might need to template more than this to be able to drop in eg Confluent's ZooKeeper Docker image though.

We're also doing some work on this locally in https://github.com/astronomerio/kafka-bundle/issues/3 and will send a PR back assuming it's successful.

**Comment by unguiculus** [comment]
I'd suggest you follow our guidelines:
https://github.com/kubernetes/charts/blob/master/REVIEW_GUIDELINES.md#configuration

**Comment by squ1d123** [comment]
Hey can this please be reviewed?
/assign @lachie83

**Comment by unguiculus** [comment]
@squ1d123 I did review and made a comment. Please follow that.

File: incubator/zookeeper/values.yaml [hunk 2]
```diff
@@ -17,6 +17,8 @@ heap: "2G"
 serverPort: 2888
 leaderElectionPort: 3888
 clientPort: 2181
+image: "gcr.io/google_samples/k8szk"
+imageTag: "v2"
 imagePullPolicy: "Always"
```
Comment (cmarquezrusso) [inline incubator/zookeeper/values.yaml:22]: pullPolicy: IfNotPresent

**Comment by ghost** [comment]
related: https://github.com/kubernetes/charts/pull/4931

**Comment by unguiculus** [comment]
Closing this in favor of #4931. Thanks.

**Pull request closed without merging**